### DISCUSSION TITLE
New FIOS html remote

### DISCUSSION THREAD

**Discussion by ghost** [GerryDazoo/Slinger#535] [General]
### Discussed in https://github.com/GerryDazoo/Slinger/discussions/512

<div type='discussions-op-text'>

<sup>Originally posted by **mdehrlich** January 27, 2023</sup>
This has some features that require changes to the sling server code to work.  
1) I can use this one remote for all 6 of my configs.  It autoselects the based on the running stream.
2) I auto insert the names of the boxes into the drop down menu.
If anyone wants to use this, I can tell you what changes are required to the server.
[FIOS_remote.zip](https://github.com/GerryDazoo/Slinger/files/10523291/FIOS_remote.zip)
</div>

**Comment by ghost** [comment]
Has anyone gotten the remote to work with FIOS ONE cable boxes? I just went to fios one and the remote no longer works.
I kept an old fios remote that uses ir and it still works, so why not the slinger remote?

**Comment by GerryDazoo** [comment]
Because, your video source hasn't been properly configured for the FIOS remote?

**Reply by ghost** [reply]
I may have been misleading in posting to this thread. I did a google search on slinger fios one and this thread popped up.
I am using the windows executable slingbox_server.exe and remote.txt. 
My .ini file has vidoe source=1. I've tried all other values and 1 is the only one that works. And the picture is good. 
I just can't change the channel or bring up the dvr or guide, etc using remote.txt anymore.

**Reply by socelillyhammer** [reply]
Ive had the problem myself and my SB was working in the past with the fios one boxes.  I ended up buying a 3rd party external device that i now use as the remote and coded the buttons using the verizon fios Pv265v5 remote. If you do figure this out let me know.
